build.sbt

```diff
@@ -450,52 +451,130 @@ lazy val scalalib =
     )
     .settings(mavenPublishSettings)
     .settings(
-      assembleScalaLibrary := {
-        import org.eclipse.jgit.api._
-
-        val s      = streams.value
-        val trgDir = target.value / "scalaSources" / scalaVersion.value
-        val scalaRepo = sys.env.getOrElse("SCALANATIVE_SCALAREPO",
-                                          "https://github.com/scala/scala.git")
-
-        if (!trgDir.exists) {
-          s.log.info(
-            s"Fetching Scala source version ${scalaVersion.value} from $scalaRepo")
-
-          // Make parent dirs and stuff
+      // code to fetch scala sources adapted, with gratitude, from
+      // Scala.js Build.scala, lines 1125 to 1233, at the suggestion of
+      // @sjrd.
+      //
+      // By intent, the content is intended to be as similar as feasible.
+      // The forced differences are:
+      //   1) Scala Native build.sbt uses a slightly different baseDirectory
+      //      than Scala.js. See commented starting with "SN Port:" below.
+      //      This is the only intended change with meaning between the two.
+      //   2) This file is sbt, Scala.js uses a .scala file. Trailing commas
+      //      may differ.
+      //   3) The project scalafmt configuration differs between Scala Native
+      //      and the Scala.js so there are annoying but not meaningful
+      //      format differences.
```

Comment:
While 1) is clearly interesting going forward, 2) and 3) are distraction for the next person reading this code. Differences in format between Scala.js and Scala Native are *everywhere*, not just in this specific spot, so pointing it out here is distracting.

Reply:
Done.